src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs

```diff
@@ -54,7 +54,10 @@ protected WKWebView PrepareWKWebView()
         {
             WKWebViewConfiguration wkconfg = new WKWebViewConfiguration() { };
 #if !IS_APPCENTER_BUILD
-            wkconfg.DefaultWebpagePreferences.PreferredContentMode = WKContentMode.Mobile;
+            if (UIDevice.CurrentDevice.CheckSystemVersion(13, 0))
```

Comment:
what is the default value? for different iOS versions? perhaps a note on that here or a link to get that info from apple?

Reply:
this is only applicable in ios 13+

---
In reply to: [378509016](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1627#discussion_r378509016) [](ancestors = 378509016)